src/Stryker.CLI/README.md

```diff
@@ -54,6 +54,53 @@ All available loglevels are:
 
 `dotnet stryker --max-concurrent-test-runners`
 
+#### Mutation testrun reporters 
+
+`dotnet stryker --reporters ['ConsoleReport', 'ConsoleProgressBar']`
+
+All available reporter options are:
+ * All - This will enable all available reporters
+ * ConsoleProgressBar - This reporter will display the progress of the mutation testrun as a progress bar
+ Example output:
```

Comment:
`Example output:` could use an extra newline. Now it is displayed right behind the ConsoleProgressBar explaination.

```suggestion

Example output:
```